docs/06_how-to-guides/30_key-value-api/kv_map/90_how-to-allow-users-to-pay-kv-map.md

```diff
@@ -3,18 +3,18 @@ content_title: How-To Allow Users to Pay for Key-Value Map Resources
 link_text: "How-To Allow Users to Pay for Key-Value Map Resources"
 ---
 
-## Summary
+## Overview
 
-This how-to procedure provides instructions to allow users to pay for the resources needed to store data in a `Key-Value Map` (`kv map`).
+This how-to provides instructions to allow users to pay for the resources needed to store data in a `Key-Value Map` (`kv map`).
```

Comment:
This guide provides instructions which show how you ... set up/write code in/define  ... a smart contract which requires users to pay for the resources used to store data in a `Key-Value Map` (`kv map`).

Reply:
allow seems wrong as I think you are 'forcing' users to pay for the resources

Reply:
I am missing your point, I do not see how "allow someone" is leading the thought to "forcing someone"...
the title of the how to is also "how to allow users to pay for..."

Reply:
Does the user have a choice in who pays? "allow users to pay" gives me the impression that they have a choice.

Reply:
perhaps I'm wrong but I think the code 'forces' the users to pay, rather than the smart contract.

Reply:
'forces' is in quotes ... using 'forces' is an implied way of making the point - which is the user has to pay.

Reply:
So back to the question: Are users allowed to pay or do they have to pay?

Reply:
someone has to pay. 
and normally the payer is the account that owns the smart contract.

however, this tutorial, is showing how the code (the smart contract) can allow someone else to pay, in this particular case the user that upserts is the payer for the RAM needed to store its row/data.

this is good churning.

are these better titles for the how to?

How to explicitly appoint a payer
How to set the payer

Reply:
How to set up an action which requires the user to pay.
How to require users pay for their actions.

Reply:
I went with the first! thanks!